tests/evaluation/test_stochastic.py

```diff
@@ -115,32 +115,4 @@ def test_groupby_avg(self):
         assert(dp_res == True)
         assert(acc_res == True)
```

Comment:
These checks a can all be:

 assert (acc_res) 

just fyi

Reply:
Done